core/src/test/java/io/grpc/internal/ManagedChannelImplIdlenessTest.java

```diff
@@ -205,15 +204,14 @@ public void enterIdleModeAfterForceExit() throws Exception {
     newTransports.drainTo(transports);
     assertEquals(addressGroupList.size(), transports.size());
 
-    InterimTransport<ClientTransport> interimTransport = channel.tm.createInterimTransport();
+    channel.tm.createInterimTransport();
```

Comment:
These tests aren't sharing that view with you. They don't shutdown anything. It's not this one line then that would need to be changed, but the entire file.